staging/src/k8s.io/apiserver/pkg/endpoints/handlers/rest_test.go

```diff
@@ -347,7 +347,7 @@ func (tc *patchTestCase) Run(t *testing.T) {
 			name,
 			patchType,
 			patch,
-			namer, creater, defaulter, convertor, kind, resource, codec)
+			namer, creater, defaulter, convertor, kind, resource, codec, nil)
```

Comment:
So I would create one here instead of passing nil.

Reply:
Done.